web/auth/auth.go

```diff
@@ -9,8 +9,8 @@ import (
 	pb "github.com/autograde/aguis/ag"
 	"github.com/autograde/aguis/database"
 	"github.com/jinzhu/gorm"
-	"github.com/labstack/echo"
 	"github.com/labstack/echo-contrib/session"
+	echo "github.com/labstack/echo/v4"
```

Comment:
I don't think the `echo` alias is necessary; it is used in some files, but not in others.

Reply:
I mean, I think it will use `echo` as the package instead of `v4`...

Reply:
Fixed.